portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,183 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <!-- <meta content ="width=device-width, initial-scale=1.0"> -->
+    <title >Portfolio  </title>
+    <link rel="stylesheet" href="boostrap-grid.css"  type="text/css">
+     <link rel="stylesheet" href="style.css"  type="text/css">
+     <link rel="preconnect" href="https://fonts.gstatic.com">
+     <link rel="preconnect" href="https://fonts.gstatic.com">
+     <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&family=Inknut+Antiqua:wght@300&display=swap" rel="stylesheet">
+   
+    <script src="https://use.fontawesome.com/d1341f9b7a.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <body>
+
+    <nav >
+        <ul>
+      <li><a href="#contact">contact</a></li>
+     <li><a href="#projects">projects</a></li>
+        <li><a href="#experience ">experience</a></li>
+        <li class="empty"></li>
+        <li><a href="https://twitter.com/HumanCyyborg" target="_blank">twitter</a></li>
+       <li><a href="https://www.linkedin.com/in/danieldawsonkalu/" target="_blank">Linkedin</a></li>
+        
+    </ul>
+
+    </nav>
```

Comment:
Super nit: for the future, it's good to have some linter/code cleaner to keep things nice and formatted so it's easier to read.